source/includes/_tipser-script.md

```diff
@@ -71,12 +71,14 @@ Complete working example could look like:
     <div id="tipser_store"></div>
     <script src="https://cdn.tipser.com/tipser-script/latest.js"></script>
     <script>
-      tipser.elements("posId");
+      tipser.elements("posId", {lang: "en-US", openOldDialog: "false"});
     </script>
   </body>
 </html>
 ```
 
+<aside class="info">We recommend every new implementation of Tipser Script to enable new modal dialogs implementation by setting <code>openOldDialog</code> to <code>false</code>, as in the snippet above. (In the next major release, the default implementation will be active by default)</aside>
```

Comment:
In the next major release, the default implementation will be active by default => 
In the next major release, **this** implementation will be active by default